artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/QueueImpl.java

```diff
@@ -133,7 +133,7 @@
    protected static final int CRITICAL_CONSUMER = 3;
    protected static final int CRITICAL_CHECK_DEPAGE = 4;
 
-   private static final Logger logger = Logger.getLogger(QueueImpl.class);
+   protected static final Logger logger = Logger.getLogger(QueueImpl.class);
```

Comment:
Should keep private and if a subclass wants to log then it makes its own logger , so its logged against the right class

Reply:
fixed